modules/tinymce/src/plugins/table/main/ts/ui/MenuItems.ts

```diff
@@ -36,46 +36,120 @@ const addMenuItems = (editor: Editor, selectionTargets: SelectionTargets) => {
     onAction: cmd('mceTableDelete')
   };
 
-  const rowItems = [
-    { type: 'menuitem', text: 'Insert row before', icon: 'table-insert-row-above', onAction: cmd('mceTableInsertRowBefore'), onSetup:  selectionTargets.onSetupCellOrRow },
-    { type: 'menuitem', text: 'Insert row after', icon: 'table-insert-row-after', onAction: cmd('mceTableInsertRowAfter'), onSetup:  selectionTargets.onSetupCellOrRow },
-    { type: 'menuitem', text: 'Delete row', icon: 'table-delete-row', onAction: cmd('mceTableDeleteRow'), onSetup:  selectionTargets.onSetupCellOrRow },
-    { type: 'menuitem', text: 'Row properties', icon: 'table-row-properties', onAction: cmd('mceTableRowProps'), onSetup:  selectionTargets.onSetupCellOrRow },
-    { type: 'separator' },
-    { type: 'menuitem', text: 'Cut row', onAction: cmd('mceTableCutRow'), onSetup:  selectionTargets.onSetupCellOrRow },
-    { type: 'menuitem', text: 'Copy row', onAction: cmd('mceTableCopyRow'), onSetup:  selectionTargets.onSetupCellOrRow },
-    { type: 'menuitem', text: 'Paste row before', onAction: cmd('mceTablePasteRowBefore'), onSetup:  selectionTargets.onSetupCellOrRow },
-    { type: 'menuitem', text: 'Paste row after', onAction: cmd('mceTablePasteRowAfter'), onSetup:  selectionTargets.onSetupCellOrRow }
-  ] as Array<Menu.NestedMenuItemContents>;
+  editor.ui.registry.addMenuItem('TableInsertRowBefore', {
```

Comment:
All the names should be in lowercase fo consistency, as currently we have the following:
- tableprops
- row
- cell
- deletetable
- inserttable

- [ ] Convert menu names to lowercase